src/Microsoft.DotNet.Darc/DarcLib/Helpers/ManifestHelper.cs

```diff
@@ -65,7 +71,8 @@ public class ManifestHelper
                         new
                         {
                             name = asset.Asset.Name,
-                            origin = assetOriginMap.TryGetValue(asset.Asset.Name, out var origin) ? origin : null,
+                            origin = assetReleaseMetadataMap.TryGetValue(asset.Asset.Name, out var data) ? data.Origin : null,
+                            dotnetReleaseShipping = data != null && data.DotNetReleaseShipping,
```

Comment:
```suggestion
                            dotnetReleaseShipping = data?.DotNetReleaseShipping,
```
does this work? not sure

Reply:
I don't know, for me it's really hard to understand this syntax

Reply:
Then keep it, it's not a big deal at all